kickstart/kickstart.json

```diff
@@ -1,15 +1,16 @@
 {
     "variables": {
-        "applicationId": "E9FDB985-9173-4E01-9D73-AC2D60D1DC8E",
-        "apiKey": "this_really_should_be_a_long_random_alphanumeric_value_but_this_still_works_dont_use_this_in_prod",
+        "apiKey": "33052c8a-c283-4e96-9d2a-eb1215c69f8f-not-for-prod",
         "asymmetricKeyId": "#{UUID()}",
-        "clientSecret": "this_really_should_be_a_long_random_alphanumeric_value_but_this_still_works_dont_use_this_in_prod",
```

Comment:
I don't mind leaving this as the same for all sample apps.  Good to have something awkward there to people know to change.